fud/fud/main.py

```diff
@@ -52,6 +52,14 @@ def register_stages(registry, cfg):
             "Compile Calyx to Verilog instrumented for simulation",
         )
     )
+    registry.register(
+        futil.FutilStage(
+            cfg,
+            "icarus-verilog",
+            "-b verilog --disable-verify --disable-init",
+            "Compile Calyx to Verilog instrumented for simulation",
+        )
+    )
```

Comment:
Nothing to be done about it here, surely, but I suppose the existence of this "other Verilog" does sort of indicate that it would be someday nice for fud to have a more general facility for multiple routes between the same pair of types, huh?

(I'd also be interested to hear someday about why the initialization needs to be different for Icarus vs. Verilator.)

Reply:
The interpreter and icarus will now both use the `verilog.data` field instead to find out where the data is.